env.default

```diff
@@ -28,6 +28,15 @@ X_FRAME_OPTIONS=DENY
 XSS_PROTECTION=True
 XROBOTSTAG_ENABLED=False
 
+#basket
+
+# See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
+BASKET_URL=https://basket-dev.allizom.org/
+BASKET_API_KEY=
```

Comment:
```suggestion
```
I know we agreed to add this originally, but I don't think we'll ever need this in donate, so it's probably good to remove.